transport/transport-runtime/transport-runtime.gradle

```diff
@@ -34,6 +35,17 @@ android {
     }
 }
 
+firebaseTestLab {
```

Comment:
Will it be sufficient to run these on testLab when on CI, but locally otherwise?
Otherwise, we should document this as setup instructions for the project.

Reply:
Chatted with Vlad.
This will change only the deviceCheck tasks and not the connectedCheck tasks as I initially assumed.
It is indeed the case that connectedCheck can be run locally without Test Lab